sound/soc/intel/common/soc-acpi-intel-cml-match.c

```diff
@@ -95,7 +95,7 @@ static const struct snd_soc_acpi_endpoint spk_r_endpoint = {
 
 static const struct snd_soc_acpi_adr_device rt700_1_adr[] = {
 	{
-		.adr = 0x000110025D070000,
+		.adr = 0x000120025D070000,
```

Comment:
that doesn't seem right for the RT700, what I see in the DSDT is 

````
            Name (_ADR, 0x000010025D070000)  // _ADR: Address
            Name (_ADR, 0x000110025D070000)  // _ADR: Address
            Name (_ADR, 0x000210025D070000)  // _ADR: Address
````

Reply:
@bardliao  we have a cml device with RT700, you can ask keqiao for help.

Reply:
> @bardliao we have a cml device with RT700, you can ask keqiao for help.

I checked with @shumingfan and he confirmed that RT700's sdw version id is 1.